packages/gatsby/src/redux/types.ts

```diff
@@ -122,8 +122,8 @@ export interface IGatsbyState {
   pages: Map<string, IGatsbyPage>
   schema: GraphQLSchema
   status: {
-    plugins: {}
-    PLUGINS_HASH: Identifier
+    plugins: Record<string, IGatsbyPlugin>
+    PLUGINS_HASH?: Identifier
```

Comment:
Had to update `PLUGINS_HASH` as optional in order to match the default state described in `redux/reducers/status.ts` on line 5.

Reply:
I think it would be better to add PLUGINS_HASH as an empty string in the reducer init. It's easier to write code against values rather than against existence and values.

Reply:
Just pushed a new version reverting this change :)